packages/govuk-frontend-review/src/common/middleware/recolour.mjs

```diff
@@ -0,0 +1,38 @@
+import express from 'express'
+
+const router = express.Router()
+
+const RECOLOUR_COOKIE_NAME = 'use-recolour'
+
+router.use((req, res, next) => {
+  const { cookies } = req
+
+  res.locals.useRecolourClass = cookies?.[RECOLOUR_COOKIE_NAME] === 'yes'
+  res.locals.pizza = cookies
```

Comment:
🍕 😆

Reply:
Lmao whoops. I've exposed how I do my local troubleshooting.

Reply:
I figured 😆 Wondering if we should just expose `req` and `res` as locals by default, for troubleshooting and save us that kind of hiccups 😁